terraform/backend_xi.tf

```diff
@@ -0,0 +1,56 @@
+module "backend_xi" {
+  source = "git@github.com:trade-tariff/trade-tariff-platform-terraform-modules.git//aws/ecs-service?ref=aws/ecs-service-v1.5.0"
+
+  service_name  = var.service_name
```

Comment:
We'll need to extend this for each service, otherwise there will be a naming clash. `${var.service_name}-xi` and `${var.service_name}-uk` respectively?